### ISSUE TITLE
Hitting tab inside InlineEdit 'closes' input

### ISSUE THREAD

**Issue by iowillhoit** [salesforce/design-system-react#702]
Seems to be an accessibility issue. You should be able to hit tab to focus on the (x) icon, and then hit space to actually 'close' the input.

![2016-10-05 15 20 30](https://cloud.githubusercontent.com/assets/1715111/19130123/8cd08694-8b0f-11e6-8741-0e9b5058be79.gif)

**Comment by interactivellama** [comment]
@donnieberg I think this is a won't fix. What's a11y team take on this?

**Comment by donnieberg** [comment]
@interactivellama actually, this is something that I'm changing in the Inline Edit component. The new behavior will be that when you're focused in the inline edit, you can tab to the "remove" icon, and then the next tab will blur the input. I posted about this change in the DSR slack channel and folks seems to be cool with the change.

**Comment by donnieberg** [comment]
FYI @arodenbeck, I submitted a PR to change the behavior of Inline Edit so that you can Tab to the close button when in edit mode. When your focus is on the close button, pressing Tab again will blur the field now: https://github.com/salesforce/design-system-react/pull/1332
